src/github/queries.gql

```diff
@@ -270,6 +290,63 @@ query PullRequest($owner: String!, $name: String!, $number: Int!) {
 	}
 }
 
+query Issue($owner: String!, $name: String!, $number: Int!) {
+	repository(owner: $owner, name: $name) {
+		pullRequest: issue(number: $number) {
+			number
+			url
+			state
+			body
+			bodyHTML
+			title
+			author {
+				login
+				url
+				avatarUrl
+			}
+			createdAt
+			updatedAt
+			labels(first: 50) {
+				nodes {
+					name
+				}
+			}
+			id
+			databaseId
+		}
+	}
+	rateLimit {

```

Comment:
we query for the rate limit in a lot of places but then ultimately don't use this information, this can be removed

Reply:
Removed from all new queries.